api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiPatient.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+import org.openmrs.Patient;
+
+/**
+ * Defines a MpiPatient in the system. A mpi patient is simply an extension of a patient and all that that
+ * implies.
+ */
+public class MpiPatient extends Patient {
+
+    private static final boolean mpiPatient = true;
```

Comment:
It is more clear to use a constant for:
* things that would be hard to read inline, like QUERY_URL or MILLISECONDS_PER_DAY
* things that may change, like IS_XYZ_ENABLED or NUMBER_OF_TIMES

However in this case the entire purpose of the MpiPatient class is to say that this always true, so personally I think it makes things less clear.

Also, you didn't capitalize the constant name, which is probably why I looked at it in the first place.

(Regardless I think that we should change the implementation here, which would make these comments irrelevant.)